src/ui/index.ts

```diff
@@ -1,26 +1,27 @@
 export * from './Avatar/Avatar';
 export * from './Box/Box';
 export * from './Button/Button';
+export { default as CenteredBox } from './CenteredBox/CenteredBox';
```

Comment:
`CenteredBox` for some reason was implemented as a default rather than named export, hence why we'd need to do it this way. We could just refactor to be a named export, I just wasn't sure and left as it was.